src/vm/prestub.cpp

```diff
@@ -269,6 +275,11 @@ PCODE MethodDesc::MakeJitWorker(COR_ILMETHOD_DECODER* ILHeader, CORJIT_FLAGS fla
          GetMethodTable()->GetDebugClassName(),
          m_pszDebugMethodName));
 
+#if defined(FEATURE_JIT_PITCHING)
+    bool fDoJitPitching = flags.IsSet(CORJIT_FLAGS::CORJIT_FLAG_DO_PITCHING);
+    CheckStacksAndPitch(fDoJitPitching);
```

Comment:
Just a ping that this is still an open issue. Feel free to resolve it at whatever pace works for you.